config/packages/easy_admin/participation.yaml

```diff
@@ -7,6 +7,7 @@ easy_admin:
                 list: 'easy_admin/Participation/list.html.twig'
             list:
                 filters: ['conference', 'participant']
+                item_permission: ['OWNER', 'ROLE_ADMIN']
```

Comment:
Let's encapsulate the decision about ROLE_ADMIN directly in the final voter.
This will avoid to scatter all permissions in the application